pbspro.spec.in

```diff
@@ -140,6 +146,29 @@ job scheduler designed to improve productivity, optimize
 utilization & efficiency, and simplify administration for
 HPC clusters, clouds and supercomputers.
 
+%if %{with ptl}
+%package %{pbs_ptl}
+Summary: PBS ptl suite for testing
+Group: System/Base
+Requires: python-nose
+Requires: python-beautifulsoup
+%if 0%{?rhel} 
+Requires: pexpect
+%else
+Requires: python-pexpect
+%endif
+Requires: python-defusedxml
+
+%description %{pbs_ptl}
+PBS Professional® is a fast, powerful workload manager and
+job scheduler designed to improve productivity, optimize
+utilization & efficiency, and simplify administration for
+HPC clusters, clouds and supercomputers.
+
+This package is intended for testing PBS.
```

Comment:
Remove the PBS Professional description and replace the last line with:

PBS Test Lab is a test harness and test suite intended to validate the
functionality of PBS Professional®.

Reply:
PTL isn't just a test harness.  It is a python interface to PBS.  I don't think we should have its explanation as just about testing.